addons/web_editor/controllers/main.py

```diff
@@ -173,7 +173,16 @@ def attachment_update(self, attachment, name=None, width=0, height=0, quality=0,
         if attachment.type == 'url':
             raise UserError(_("You cannot change the quality, the width or the name of an URL attachment."))
         if copy:
+            original = attachment
             attachment = attachment.copy()
+            attachment.original_id = original
+            # Uniquify url by adding a path segment with the id before the name
+            if attachment.url:
+                url_fragments = attachment.url.split('/')
+                url_fragments.insert(-1, str(attachment.id))
+                attachment.url = '/'.join(url_fragments)
```

Comment:
Should not this 6-lines part be a fix in 13.0 in fact? Or maybe copying an URL attachment is a new feature?

Reply:
You still cannot copy URL attachments. This is for the special cases where an attachment is binary AND has an URL, which at the moment is only the case of unsplash images, and yes, optimizing unsplash images is a new feature.

Reply:
:+1